src/org/labkey/test/tests/DataReportsTest.java

```diff
@@ -547,8 +527,6 @@ public void testRReportShowSource()
 
         impersonateRole("Reader");
         clickFolder(getFolderName());
-
-        navigateToFolder(getProjectName(), getFolderName());
```

Comment:
I'm assuming this line and the goToProjectHome line below were removed because they are redundant. Is that correct?

Reply:
Correct.